### DISCUSSION TITLE
How to use AtomicDesignPageRenderer?

### DISCUSSION THREAD

**Discussion by jfreites** [PascaleBeier/filament-pages#7] [Q&A] [answered]
Hi,

I like to know how I can use thar kind of render. I dont have so much experience with components, but I need to have diferents layouts in my website.

Appreciate for the help!

**Comment by PascaleBeier** [comment, marked as answer]
Hey,

Both the SimplePageRenderer and the AtomicDesignRenderers are examples on how to implement your own.

Check out the comments in `filament-pages.php`: https://github.com/PascaleBeier/filament-pages/blob/286502ec2631c9ea0c0c5de5d0979efdf9c0c6f1/config/filament-pages.php#L113 to learn more:

    | If you want to use the Rendering functionality, you can create your
    | own Renderer here. Take the available Renderers for reference.
    | See FilamentPageController for recommended usage.
    |
    | Available Renderers:
    | - SimplePageRenderer:
    |   Renders everything to the defined layout below.
    | - AtomicDesignPageRenderer:
    |   More opinionated Renderer to be used with Atomic Design.
    |
    | To use the renderer, Add a Route for the exemplary FilamentPageController:
    |
    |  Route::get('/{filamentPage}', [FilamentPageController::class, 'show']);

**Reply by jfreites** [reply]
Sure, I saw that, but I litte bit confused how the data is used on the views.

**Reply by PascaleBeier** [reply]
You can check out the controller or view that is used in the FilamentPageController
